catkit2/services/thorlabs_cubes/thorlabs_cubes.py

```diff
@@ -0,0 +1,108 @@
+from catkit2.testbed.service import Service
+
+import thorlabs_apt as apt
+import numpy as np
+
+
+class ThorlabsCubes(Service):

```

Comment:
The service class and file name, as well as the directory they are in all need to have the same name for things to work smoothly in catkit2, while abiding to the camel case vs. underscore notations. I would also make the name very explicit by adding "motor" to it. One option would be to use the name "Thorlabs Cube Motor", which would mean `ThorlabsCubeMotor` for the class, `thorlabs_cube_motor.py` for the file and `thorlabs_cube_motor` for the directory.